README.md

```diff
@@ -22,6 +22,38 @@ Update config.yml and run
 cp2influxdb -c config.yml
 ```
 
+### Systemd
+
+Insert this snippet to the file /lib/systemd/system/cp2influxdb.service:
+```
+[Unit]
+Description=COOPER cp2influxdb
+After=network.target
+
+[Service]
+Type=simple
+User=pi
+ExecStart=/usr/local/bin/cp2influxdb -c /etc/cooper/cp2influxdb.yml
+Restart=always
+RestartSec=5
+StartLimitIntervalSec=0
```

Comment:
It is a wrong placement `StartLimitIntervalSec` must be in the `[Unit]` section, see:

- https://www.freedesktop.org/software/systemd/man/systemd.unit.html
- https://www.freedesktop.org/software/systemd/man/systemd.service.html